product/reputation.md

```diff
@@ -9,20 +9,74 @@ Reputation cannot be "spent" or "used". However, achieving certain Reputation th
 ## Purpose
 
 The purpose of the reputation system is to:
+
 1. Maximise growth-centric interactions between students and mentors
 2. Maximise the desire of mentors to want to contribute
 
 ## Acquisition of Reputation
 
 Reputation can be acquired in a variety of ways:
+
 1. Mentoring others' solutions successfully
-2. Writing analysers
-3. Updating analysers
-4. Creating documentation
-5. Updating documentation
+2. Writing exercises
+3. Updating exercises
+4. Creating pull requests
+5. Reviewing pull requests
+6. Merging pull requests
+7. Writing analysers
+8. Updating analysers
+9. Creating documentation
+10. Updating documentation
 
 ### 1. Mentoring others' solutions successfully
+
 Algorithm for determining the allocation of reputation points per mentoring interaction is yet to be determined.
 
+### 2. Writing exercises
+
+For each exercise where the user is listed as an author, `20` reputation is awarded.
+
+### 3. Updating exercises
+
+For each exercise where the user is listed as a contributor, `10` reputation is awarded.
+
+### 4. Creating pull requests
+
+For each merged pull request that was opened by the user, `12` reputation is awarded.
+
+- If a pull request is still open, no reputation is awarded (yet).
+- If a pull request is closed _without_ merging, no reputation is awarded.
+- It is possible to award more (or less) reputation for a merged pull request by adding one of the following labels to the pull request:
```

Comment:
I couldn't come up with a nice way to say: "only use this in more exceptional cases". Any suggestions?

Reply:
```suggestion
- In exceptional circumstances (either tiny PRs changing a few lines, or large PRs that will have taken a greater than normal of effort) is possible to award more (or less) reputation for a merged pull request by adding one of the following labels to the pull request:
```